ERROR: "IndexError" exception when attempting to generate a report for {'fusion_only_gene': 'FAM22B', 'intrcn_and_fusion_gene': 'YWHAE', 'intrcn_only_gene': 'YWHAZ', 'YWHAZ': 'P63104', 'YWHAE': 'P62258'}: list index out of range